extensions/interactions/TextInput/directives/text-input-rules.service.ts

```diff
@@ -29,51 +29,56 @@ import { TextInputRuleInputs } from 'interactions/rule-input-defs';
 export class TextInputRulesService {
   constructor(private nws: NormalizeWhitespacePipe) {}
   Equals(answer: TextInputAnswer, inputs: TextInputRuleInputs): boolean {
-    var normalizedAnswer = this.nws.transform(answer);
-    var normalizedInput = this.nws.transform(inputs.x);
-    return normalizedAnswer.toLowerCase() === normalizedInput.toLowerCase();
+    const normalizedAnswer = this.nws.transform(answer).toLowerCase();
+    const normalizedInput = inputs.x.map(
+      input => this.nws.transform(input).toLowerCase());
+    return normalizedInput.includes(normalizedAnswer);
```

Comment:
```suggestion
    return inputs.x.some(input => this.nws.transform(input).toLowerCase() == normalizedAnswer);
```

Possibly simpler/more efficient alternative. If the double checking seems more complicated, then I suggest keeping the current implementation.

Reply:
Done